### ISSUE TITLE
Different authors for different posts

### ISSUE THREAD

**Issue by keilw** [jbake-org/jbake#662]
At the moment every post has just one author based on the `meta.author` property.
Would it be possible to also have an (optional) property on a post like `author=<authorname>` that is used where present and if not given, the default author from the metadata is used instead?

**Comment by jonbullock** [comment]
Yeah that's possible, you'll want to do something similar to this: https://github.com/jbake-org/jbake-example-project-freemarker/blob/master/templates/header.ftl#L5

**Comment by ge0ffrey** [comment]
Freemarker has an elvis operator (AKA defaultIfNull operator), here's how I use it in my base.ftl:

`<link content="${(content.canonical_url)!config.canonicalBaseUrl + "/" + content.uri}" rel="canonical">`
